spec/ParseQuery.spec.js

```diff
@@ -3684,6 +3684,75 @@ describe('Parse.Query testing', () => {
     });
   });
 
+  it("includeAll", (done) => {
```

Comment:
note to self: add rule on " '.  Surprised the default airbnb we use doesn't have an opinion....